source/gui/settingsDialogs.py

```diff
@@ -35,6 +35,18 @@
 	updateCheck = None
 import inputCore
 
+#Choices for numbers combo box in Voice settings.
+
+DIGIT_CHOICES = [
+	# Translators: Choice in a combo box for speaking numbers by word.
+	_("Words"),

```

Comment:
* "Controlled by synthesizer" is quite long, though. I would have thought most synths did full numbers by default. I'm also concerned some users won't really understand what this means in practice.
* That said, "Words" is potentially confusing, since double and triple digits technically produce words. Maybe we could call it "Full" or "Full numbers"?